docs/custom_mutators.md

```diff
@@ -81,6 +82,9 @@ def queue_new_entry(filename_new_queue, filename_orig_queue):
     pass
 ```
 
+def introspection():
```

Comment:
We may want to know on mutator load if introspection is enabled, so we don't have to recompile/rewrite the script?

Reply:
IMHO that is up to the user. in my libfuzzer custom mutator introspection code I made a compile target and #if in the code. do you have a better idea? introspection needs also be compiled into afl-fuzz (INTROSPECTION=1).